data/tools/FINDPY

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+{ find data/tools  -name .git -prune -o -name '*.py' -print ;
+echo "data/tools/GUI.pyw" ;
+echo "data/tools/wmlxgettext" ;
+echo "data/tools/about_cfg_to_wiki" ;
+echo "data/tools/check_mixed_indent"  ;
+echo "data/tools/extractbindings" ;
+echo "data/tools/imgcheck" ;
+echo "data/tools/steam-changelog" ;
+echo "data/tools/TeamColorizer" ;
+echo "data/tools/tmx_trackplacer" ;
+echo "data/tools/wesnoth_addon_manager" ;
+echo "data/tools/wmlflip" ;
+echo "data/tools/wmlindent" ;
+echo "data/tools/wmllint" ;
+echo "data/tools/wmllint-1.4" ;
+echo "data/tools/wmlscope" ;
+echo "data/tools/wmlunits" ;
+echo "data/tools/wmlxgettext" ;
+} | sort |
+while read file; do
+	if grep -m 1 -E " *_ *= *gettext\\.translation\\((['\"])$1\\1" $file > /dev/null 2>&1;
```

Comment:
Is there a need to do this, instead of just an unconditional `echo $file` (or just not filter at all)?

Reply:
Unless we prefer refactoring the build scripts _and_ special casing the ``wesnoth-tools`` domain, I think so.

I was actually thinking of drying the ``FINDCFG`` scripts (there is one of them, exactly the same, for each text domain), using a similar approach to this ``FINDPY`` file.

Reply:
Ah, I thought it was filtering for files that contained `gettext.translation` to filter out files that don't contain any translatable text, rather than filtering for a specific textdomain.

Please add comments about what the behavior of this script is. It's different to the `FINDCFG` scripts (which don't filter for textdomains), and reading a regex isn't documentation.

Also, instead of `-m 1` and a conditional print, grep's `--files-with-matches` option will print just the filename for you.

Reply:
Great suggestion. Applied.

Reply:
I still feel this the FINDPY file needs more comments, but the issue raised here is resolved, so I'll mark the conversation as resolved.